pkgs/games/am2r-launcher/default.nix

```diff
@@ -0,0 +1,106 @@
+{ lib
+, buildDotnetModule
+, pkgsi686Linux
```

Comment:
This can be removed.
(FYI, you can use language servers like `nil` to automatically detect unused inputs)